### PR TITLE
Add notes for CUDA 12.2 on Windows support

### PR DESCRIPTION
**Pull request by kmaehashi**
https://github.com/cupy/cupy/pull/7758#issuecomment-1665202197

### CODE REVIEW

**Review by emcastillo — APPROVED** [review]

File: docs/source/install.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Requirements
 
 * `NVIDIA CUDA GPU <https://developer.nvidia.com/cuda-gpus>`_ with the Compute Capability 3.0 or larger.
 
-* `CUDA Toolkit <https://developer.nvidia.com/cuda-toolkit>`_: v11.2 / v11.3 / v11.4 / v11.5 / v11.6 / v11.7 / v11.8 / v12.0 / v12.1 / v12.2
+* `CUDA Toolkit <https://developer.nvidia.com/cuda-toolkit>`_: v11.2 / v11.3 / v11.4 / v11.5 / v11.6 / v11.7 / v11.8 / v12.0 / v12.1 / v12.2 [#cuda112windows]_
```
Comment (emcastillo) [inline docs/source/install.rst:9]: image
This is the rendered doc, I just give it another thought and maybe it is better to specify (linux only)?
I think it is not very important but just a quick note :)

Comment (kmaehashi) [reply docs/source/install.rst:9]: The point is that CUDA 12.2 for Windows is totally fine for most users with CUDA Toolkit installed, except for environments without headers...

Comment (kmaehashi) [reply docs/source/install.rst:9]: It seems only supported footnote syntax is numbers (1, 2, 3, ...) or symbols (`*`, `✝️`, ...)
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#footnotes

image

Comment (emcastillo) [reply docs/source/install.rst:9]: I guess it is fine, it can be seen at a glance so no prob!

**Pull request merged**